internal/databases/postgres/daemon_handler.go

```diff
@@ -19,6 +19,10 @@ const (
 	SdNotifyWatchdog = "WATCHDOG=1"
 )
 
+var (
+	SocketWriteFailed = "socket write failed: %w"
```

Comment:
Let's convert it to a separate error like this one https://github.com/wal-g/wal-g/blob/61e2466e2651604f5280f6742e4b29a0425aa11b/internal/fetch_helper.go#L19